spec/sidekiq/form526_status_polling_job_spec.rb

```diff
@@ -139,7 +140,7 @@
         Form526StatusPollingJob.new.perform
 
         expect(delivered_backup_submission_a.reload.aasm_state).to eq 'finalized_as_successful'
-        expect(delivered_backup_submission_b.reload.aasm_state).to eq 'finalized_as_successful'
+        expect(delivered_backup_submission_b.reload.aasm_state).to eq 'delivered_to_backup'
```

Comment:
Am i understanding correctly that `delivered_to_backup` and `rejected_by_backup` are place holders for our new pass / fail paradigm?

Reply:
This was simply to account for the fact that a response of 'success' does not imply a submission has been successfully processed and delivered to its final destination. Only a status of 'vbms' indicates a submission was successfully uploaded into a Veteran's eFolder within VBMS. This spec will change with the adoption of the new model, and the `backup_submitted_claim_status` will be evaluated, with values of `nil`, `rejected` or `accepted`, will be evaluated in place of `aasm_state`.